locales/sr_RS/LC_MESSAGES/duckduckgo.po

```diff
@@ -165,16 +166,16 @@ msgstr "Оглас је сумњив"
 
 #. From Home page, options to "Add DuckDuckGo" to browser or set as home page.
 msgid "Add DuckDuckGo"
-msgstr "Dodaj DuckDuckGo"
+msgstr "Додај ДакДакГо"
```

Comment:
Unless there is a strong reason to do this, we don't usually accept translations or transliterations for "DuckDuckGo" since that is the trade mark.

Reply:
This is simply transliteration because in Serbian grammar foreign names should be transliterated (and spelled the way they should be pronounced, Serbian is strictly phonetic language). However, if it's a trade mark issue, I can revert it back to original name.

Reply:
Hi @miloske85 - really sorry for the delay in responding. Since 'DuckDuckGo' is a registered trade mark, the usual advice is not to translate it. This is in line with what I've seen on other sites featuring a trademark.

Many thanks!

Reply:
Reverted to 'DuckDuckGo'.

Thanks.